src/pu-main.c

```diff
@@ -45,9 +50,14 @@ static GOptionEntry option_entries[] = {
         &arg_config, "Layout configuration file in YAML format", "CONFIG" },
     { "debug", 'd', G_OPTION_FLAG_NONE, G_OPTION_ARG_NONE,
         &arg_debug, "Print debug messages", NULL },
+    { "debug-domains", 'D', G_OPTION_FLAG_NONE, G_OPTION_ARG_STRING,
+        &arg_debug_domains, "Comma separated list of modules to enable debug output for",
+        "DEBUG-DOMAINS" },
```

Comment:
Use underscores instead of dashes for capitalized names:
```suggestion
        "DEBUG_DOMAINS" },
```